pylint/config/caching.py

```diff
@@ -0,0 +1,69 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+"""Caching functions.
+
+These are in pylint.config for compatibility reasons.
+"""
+
+from __future__ import annotations
+
+import pickle
+import sys
+import warnings
+from pathlib import Path
+
+from pylint.constants import PYLINT_HOME
+from pylint.utils import LinterStats
+
+
+def _get_pdata_path(
+    base_name: str | Path, recurs: int, pylint_home: str | Path = PYLINT_HOME
+) -> Path:
+    base_name = "_".join(str(p) for p in Path(base_name).parents)
+    return Path(pylint_home) / f"{base_name}{recurs}.stats"
+
+
+def load_results(
+    base: str, pylint_home: str | Path = PYLINT_HOME
+) -> LinterStats | None:
+    data_file = _get_pdata_path(base, 1, pylint_home)
+    try:
+        with open(data_file, "rb") as stream:
+            data = pickle.load(stream)
+            # TODO Remove in 3.0 # pylint: disable=fixme
```

Comment:
```suggestion
            # TODO: 3.0: Remove deprecated if statement # pylint: disable=fixme
```

I have an VS Code extension which allows ordering and filtering `TODOs` in a codebase. Using some form of unity (`TODO: version: ...`) helps doing so 😄

Reply:
Yeah sorry, there's an issue in pylint to allow TODOes with a peculiar structure I think. (issue in parenthesis maybe)